docs/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,329 @@
+
+==== internationalisation
+
+// ref ap UG, not checked
+{ap} has built-in .conf files so that it can translate captions just by setting `-a lang=<language code>`. 
+See <<language-support>> for the {ad} equivalent.
+
+==== Configuration files
+
+{ad} does not use .conf files or filters, so, `--conf-file`, `-dump-conf`, and `--filter` are not implemented. 
+
+==== Themes
+
+// ref ap UG. Actually seems to set JS as well. OK to ignore?
+{ap} themes are just stylesheets, so replace `--theme` with `-a stylesheet=<stylesheet>`.
+
+==== Doctest
+
+// ref ap UG. 
+{ap} `--doctest` runs units tests. 
+See GitHub for how to run the {ad} unit tests.
+
+==== Default html backend
+
+{ap} used xhtml 1.1 as its defualt backend; {ad} uses html5. 
+Use html5; it has much better browser support than xhtml 1.1.
+
+=== Changed syntax
+
+// Simplified from migration guide
+These changes are non backward-compatible, but if you set the the `compat-mode` attribute, {ad} will accept the {ap} syntax.
+For the long term you should convert to the {ad} syntax. 
+However if you choose to stay with compat-mode, read the {url-migrate}[Migration Guide] for the full details.
+
+[cols="1a,1a,1a"]
+|====
+|Feature |{ap} or {ad} with `compat-mode` |{ad}
+
+|_italic text_
+|pass:['italic text']
+|pass:[_italic text_]
+
+|`monospaced text`
+|pass:[+monospaced text+]
+|pass:[`monospaced text`]
+
+|Prevent attribute replacement
+|pass:[`{asciidoc-version}`]
+|pass:[`+{asciidoc-version}+`]
+
+|"`double quotes`"
+|pass:[``double quotes'']
+|pass:["`double quotes`"] or enter them directly
+
+|'`single quotes`'
+|pass:[`single quotes']
+|pass:['`single quotes`'] or enter them directly
+
+|Document title ^[1]^
+|
+[source]
+----
+Title
+=====
+----
+|
+
+[source]
+----
+= Title
+----
+
+|====
+
+//Dan calls this "setext" style, but this term is not used in either the ad or ap manuals, so I prefer to use the ap term "two line title"
+CAUTION: [1] {ad} accepts the two-line heading style to set the document title, but using it implicitly sets `compat-mode`.
+Therefore if you want to use the new {ad} syntax, make sure to use the one-line style for the document title.
+
+These non backward-compatible changes are not affected by the `compat-mode` attribute:
+
+[cols="1a,1a,2a"]
+|====
+|Feature |{ap} |{ad}
+
+|Index terms
+|pass:[`++`] and pass:[`+++`]
+|pass:[((Sword))] and pass:[(((Sword, Broadsword, Excalibur)))]
+
+|Underlined titles
+|Underline length must match title length +/- 2 characters
+|Underline length must match title length +/- 1 character (Underlined titles are deprecated anyway)
+
+|+ifeval::[]+ 
+|Flexible
+|Compares the values of attributes only
+
+|Block delimiters
+|Delimiters do not have to match in length
+|The length of start and end delimiters for a block must match
+
+|Asciidoc cell formatting (e.g `a\|`)
+|`a` or `asciidoc`
+|`a` only
+
+|Deprecated tables
+|Yes
+|No (you don't want them anyway)
```

Comment:
I'm hesitant to mention deprecated tables as that is again something that was dropped in AsciiDoc Python itself.